### PR TITLE
Fixed Some of the Alex issues.

### PR DESCRIPTION
**Pull request by RohanDhodare**
Problem:
Had Alex issues in some files

Solution:
Replaced some words as per the recommendations given in report

Signed-off-by: Rohan Dhodare <rohan.v.dhodare@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered CONTRIBUTING.md](https://github.com/RohanDhodare/cortx-motr/blob/Alex-Issues/CONTRIBUTING.md)
[View rendered SUPPORT.md](https://github.com/RohanDhodare/cortx-motr/blob/Alex-Issues/SUPPORT.md)
[View rendered doc/Seagate-FDMI-HLD.md](https://github.com/RohanDhodare/cortx-motr/blob/Alex-Issues/doc/Seagate-FDMI-HLD.md)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -175,5 +175,5 @@ Refer to the [Motr Coding Style Guide](../dev/doc/coding-style.md) and the [CORT
 
 You can reach out to us with your questions, feedback, and comments through our CORTX Communication Channels:
 
-- Join our CORTX-Open Source Slack Channel to interact with your fellow community members and gets your questions answered. [![Slack Channel](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?)
+- Join our CORTX-Open Source Slack Channel to interact with your folk community members and gets your questions answered. [![Slack Channel](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?)
```
Comment (rkothiya) [inline CONTRIBUTING.md:178]: I think we can change it to :
join our CORTX-Open Source Slack Channel to interact with other community members and get your questions answered

**Comment by gauravchaudhari02** [comment]
ok to test

File: doc/faq.rst [hunk 2]
```diff
@@ -43,6 +43,6 @@ Mero -> Motr rename
         make[1]: *** [all-recursive] Error 1
         make: *** [all] Error 2
 
-   A: Remove ``/etc/ld.so.conf.d/mero.conf``, then rebuild Motr after ``git
+   A: Remove ``/etc/ld.so.conf.d/motr.conf``, then rebuild Motr after ``git
```
Comment (nkommuri) [inline doc/faq.rst:46]: Did you verify this file exists or not?

**Comment by gauravchaudhari02** [comment]
run vm-deployment

File: doc/faq.rst (hunk 2 above)
Comment (RohanDhodare) [reply doc/faq.rst:46]: No, I will check and do the appropriate changes needed.

**Review by anaghadeshmukh — APPROVED** [review]

**Comment by rkothiya** [comment]
Giving exception for pre-merge and ci/cd as its documentation change.

**Pull request merged**